clients/ui/frontend/src/app/pages/modelCatalog/utils/__tests__/modelCatalogUtils.spec.ts

```diff
@@ -302,3 +305,178 @@ describe('filtersToFilterQuery', () => {
   //     });
   //   });
 });
+
+describe('hasFiltersApplied', () => {
+  const mockFormData = ({
+    tasks = [],
+    license = [],
+    provider = [],
+    language = [],
+    hardware_type = [],
```

Comment:
These camelcase warnings can be ignored, if you like you can throw a `/* eslint-disable camelcase */` at the top of this file.